### PR TITLE
Update the readme with PDS instructions

### PR DESCRIPTION
**Pull request by wynxel**
Updating the readme with the instractions about how we use the image.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
...
+
+## Image origin
+
 FORKED FROM: https://github.com/avencera/postgres-backup-restore that was FORKED FROM: https://github.com/schickling/dockerfiles/
+
+## Usage in the PDS
+
+Remember that this is a public repo.
+We are using only `backup` image in our infrastructure. The `restore` image is supposed to be
+used only in extraordinary situation.
```
Comment (holubales) [inline README.md]: I wouldn't use "extraordinary", we actually have a "restore" runbook which we can link here and be specific about this.

File: README.md [hunk 2]
```diff
@@ -1 +1,16 @@
...
+In order to get a new `backup` image to the PDS infrastructure, you have to
+locally build a docker image and push it to a `postgres-backup-restore`
```
Comment (holubales) [inline README.md:13]: Again, be specific. We have `make docker-build` and `make docker-push`.

File: README.md [hunk 3]
```diff
@@ -1 +1,16 @@
...
+
+## Usage in the PDS
+
+Remember that this is a public repo.
+We are using only `backup` image in our infrastructure. The `restore` image is supposed to be
+used only in extraordinary situation.
+In order to get a new `backup` image to the PDS infrastructure, you have to
+locally build a docker image and push it to a `postgres-backup-restore` 
+image repository. Please use `backup-<SHORT SHA>` tag. The `<SHORT SHA>` is a short SHA
+hash of the master commit, from which the image was built.
```
Comment (holubales) [inline README.md]: The important part is that this repo is not linked to our `ci` setup, so you need to update the tag in `pds-infra`, I think it's worth mentioning it as well.

**Review by holubales — APPROVED** [review]
Thanks for improving this, I have a few comments about clarity of the individual steps.

File: README.md (hunk 2 above)
Comment (wynxel) [reply README.md:13]: Well, unfortunately not in this repo. The ideal thing would be to have makefile + ci here, but I wanted to do just a quick fix.

File: README.md (hunk 3 above)
Comment (wynxel) [reply README.md]: Sure, I will.

File: README.md (hunk 1 above)
Comment (wynxel) [reply README.md]: added note about that. I don't want to link a private resources here, since it is a public repo.

File: README.md (hunk 2 above)
Comment (holubales) [reply README.md:13]: There is a makefile, just not in the root folder.

Comment (wynxel) [reply README.md:13]: Thanks, fix here: https://github.com/portworx/postgres-backup-restore/pull/6

**Pull request merged**